### PR TITLE
Trovit integration

### PR DESCRIPTION
**Pull request by joaodubas**
Enable trovit integration

### CODE REVIEW

File: apps/re/lib/exporters/trovit.ex [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  defp convert_attribute(:pictures, %{images: []}) do
+    {"pictures", %{}, nil}
+  end
+
+  defp convert_attribute(:pictures, %{images: images}) do
+    {"pictures", %{}, Enum.map(images, &build_image/1)}
+  end
+
+  defp convert_attribute(:owner, _) do
+    # TODO (jpd): discover what an owner means
```
Comment (pmargreff) [inline apps/re/lib/exporters/trovit.ex]: These `TODOs` could be extracted to our task board, once there they are visible by everyone, while here they will only be remembered if someone passes by.

[the remark this answers is not in the document]
Comment (rhnonose) [reply apps/re/lib/exporters/trovit.ex]: Agree, PRs shouldn't have `TODO`s, you either do it in the PR or document outside as an improvement, it has low visibility here.

File: apps/re/lib/exporters/trovit.ex [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    {"by_owner", %{}, "-"}
+  end
+
+  defp convert_attribute(:agency, _) do
+    # TODO (jpd): discover what an agency means
+    {"agency", %{}, "-"}
+  end
+
+  defp build_url(path, param) do
+    # TODO (jpd): separate this method
```
[the remark this answers is not in the document]
Comment (joaodubas) [reply apps/re/lib/exporters/trovit.ex]: I'll open a techinical improvement for this case, there are other points were we can improve the integrations.

JPD are my initials.

File: apps/re/lib/exporters/trovit.ex (hunk 1 above)
[the remark this answers is not in the document]
Comment (joaodubas) [reply apps/re/lib/exporters/trovit.ex]: Both of you are right in this case, I must confirm these infos before merge this PR.

File: apps/re_web/test/exporters/trovit/plug_test.exs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    test "should return an empty XML", %{conn: conn} do
+      conn = get(conn, "/exporters/trovit/rs/porto-alegre")
+
+      assert response(conn, 200) == @empty_listings
+    end
+  end
+
+  describe "without city and state slugs on path" do
+    test "should return not found error", %{conn: conn} do
+      conn = get(conn, "/exporters/trovit/invalid")
```
[the remark this answers is not in the document]
Comment (joaodubas) [reply apps/re_web/test/exporters/trovit/plug_test.exs]: OK, will rewrite this one.

File: apps/re/lib/exporters/trovit.ex [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+      "picture",
+      %{},
+      [
+        {"picture_url", %{}, escape_cdata("#{@image_url}/#{filename}")},
+        {"picture_title", %{}, escape_cdata(description)}
+      ]
+    }
+  end
+
+  defp escape_cdata(value) when is_binary(value) do
```
[the remark this answers is not in the document]
Comment (joaodubas) [reply apps/re/lib/exporters/trovit.ex]: I will add this note in a technical task, to improve the exporters code.

**Comment by joaodubas** [comment]
@pmargreff fixed the issue with `TODO` notes, and update the url used in one of the tests.

**Pull request merged**